com.unity.shadergraph/Editor/Data/Util/SpaceTransformUtil.cs

```diff
@@ -371,6 +375,17 @@ public static void AbsoluteWorldToWorld(SpaceTransform xform, string inputValue,
             }
         };
 
+        internal static IEnumerable<NeededTransform> ComputeTransformRequirement(SpaceTransform xform)
+        {
+            //Implicit always needed transform
+            yield return new NeededTransform(xform.from.ToNeededCoordinateSpace(), xform.to.ToNeededCoordinateSpace());
+
+            //ViaWorld & ObjectToAbsoluteWorld are also using the intermediate world space
+            var func = k_TransformFunctions[(int)xform.from, (int)xform.to];
+            if (func == ViaWorld || func == ObjectToAbsoluteWorld)
+                yield return new NeededTransform(xform.from.ToNeededCoordinateSpace(), NeededCoordinateSpace.World);
```

Comment:
should we return both (From --> World) and (World --> To) as required transforms for ViaWorld?

Reply:
Fixed at 65a242ca48740b12866a86e87fbae5ca0f8b5873